### PR TITLE
Add shortUnit JavaScript function

### PR DESCRIPTION
**Pull request by joshuaruesweg**

### CODE REVIEW

File: wcfsetup/install/files/js/WoltLabSuite/Core/StringUtil.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /**
  * Provides helper functions for String handling.
  * 
- * @author	Tim Duesterhus
+ * @author	Joshua Ruesweg, Tim Duesterhus
```
Comment (TimWolla) [inline wcfsetup/install/files/js/WoltLabSuite/Core/StringUtil.js]: Alphabetical order by last name, please

File: wcfsetup/install/files/js/WoltLabSuite/Core/StringUtil.js [hunk 2]
```diff
@@ -96,8 +96,45 @@ define(['Language', './NumberUtil'], function(Language, NumberUtil) {
...
-		unescapeHTML: function (string) {
+		unescapeHTML: function(string) {
 			return String(string).replace(/&amp;/g, '&').replace(/&quot;/g, '"').replace(/&lt;/g, '<').replace(/&gt;/g, '>');
+		},
+		
+		/**
+		 * Shortens numbers larger than 1000 by using unit prefixes.
+		 *
+		 * @param	{?}		number
+		 */
```
Comment (TimWolla) [inline wcfsetup/install/files/js/WoltLabSuite/Core/StringUtil.js]: Missing `@return`

File: wcfsetup/install/files/js/WoltLabSuite/Core/StringUtil.js [hunk 3]
```diff
@@ -96,8 +96,45 @@ define(['Language', './NumberUtil'], function(Language, NumberUtil) {
...
+			}
+			else if (number >= 1000) {
+				number /= 1000;
+				
+				if (number > 10) {
+					number = Math.floor(number);
+				}
+				else {
+					number = NumberUtil.round(number, -1);
+					console.log(number);
```
Comment (TimWolla) [inline wcfsetup/install/files/js/WoltLabSuite/Core/StringUtil.js]: Debug Code

File: wcfsetup/install/files/js/WoltLabSuite/Core/StringUtil.js [hunk 4]
```diff
@@ -96,8 +96,45 @@ define(['Language', './NumberUtil'], function(Language, NumberUtil) {
...
+				}
+				
+				unitSuffix = 'k';
+			}
+			
+			return this.formatNumeric(number, -1) + unitSuffix;
```
Comment (TimWolla) [inline wcfsetup/install/files/js/WoltLabSuite/Core/StringUtil.js]: Is the `-1` necessary at this place? It's missing in the PHP version of `shortUnit`.

Comment (joshuaruesweg) [inline wcfsetup/install/files/js/WoltLabSuite/Core/StringUtil.js]: My fault, it is not.

**Comment by joshuaruesweg** [comment]
Issues fixed 👍

**Comment by TimWolla** [comment]
Thanks.

**Pull request merged**